activesupport/lib/active_support/duration.rb

```diff
@@ -86,7 +93,7 @@ def eql?(other)
       Duration === other && other.value.eql?(value)
     end
 
-    def hash
+    def hash # :nodoc:
```

Comment:
i don't agree, this method returns hash of object like as [`Object#hash`](http://ruby-doc.org/core-2.2.2/Object.html#method-i-hash)
```
2.2.2 :067 >  1.year.hash
 => -68723588783739193
```